fss/src/app/children/child-details.component.ts

```diff
@@ -114,9 +121,10 @@ export class ChildDetails implements OnInit, OnChanges {
         if ( ! this.child.equals(this.orig_child) ) {
             if (! this.child.equals(this.changed_child)) {
                 this.changed_child = Object.assign(Object.create(this.child), this.child); // deep copy
+                this.changed_child.birth_date = this.child.birth_date; // deep copy didn't work for null/empty on this field.
                 this.unsaved = true;
             } else {
-                this.saveChild();
+                //this.saveChild();
```

Comment:
Do we still want to save here?

Reply:
These are intentional. Autosaving was causing problems so I turned it off by commenting these out.
The check for if the page data is "dirty" is still happening, and being used to make the save button pulsate.

Reply:
Ah. It's all coming together for me now.

Not totally removing it for now in case we want to turn it back on? Maybe we can add a flag somewhere to turn autosave on and off and do if checks instead of commenting out?